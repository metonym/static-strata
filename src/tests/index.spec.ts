import fs from 'fs';
import path from 'path';
import { bufferToJson, formatJson } from '../';

describe('setup', () => {
  test('formatJson', () => {
    expect(formatJson({ key: 'value' })).toMatchSnapshot();
  });

  test('bufferToJson', () => {
    const pathToPkg = path.join(process.cwd(), 'package.json');
    const buffer = fs.readFileSync(pathToPkg);
    expect(bufferToJson(buffer)).toEqual(expect.any(Object));
  });
});
