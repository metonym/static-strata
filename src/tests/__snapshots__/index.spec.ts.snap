// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`setup formatJson 1`] = `
"{
  \\"key\\": \\"value\\"
}"
`;
